pages/registries.py

```diff
@@ -14,17 +14,20 @@ class BaseRegistriesPage(OSFBasePage):
     # Components
     navbar = ComponentLocator(RegistriesNavbar)
 
+
 class RegistriesLandingPage(BaseRegistriesPage):
     url = settings.OSF_HOME + '/registries'
 
     identity = Locator(By.CSS_SELECTOR, '._RegistriesHeader_3zbd8x', settings.LONG_TIMEOUT)
-    search_button = Locator(By.CSS_SELECTOR, '[data-test-search-button]')
+    search_box = Locator(By.ID, 'search')
+
 
 class RegistriesDiscoverPage(BaseRegistriesPage):
     url = settings.OSF_HOME + '/registries/discover'
 
     identity = Locator(By.CSS_SELECTOR, '[data-test-share-logo]')
     loading_indicator = Locator(By.CSS_SELECTOR, '.ball-scale')
+    osf_filter = Locator(By.CSS_SELECTOR, '[data-test-source-filter-id="test_OSF Registries"]')
```

Comment:
I tried testing this locally. It failed b/c I was testing against staging1.  I updated the `CSS_SELECTOR` to do `endswith`-matching via: `'[data-test-source-filter-id$="_OSF Registries"]'`, and that seems to fix the issue.  I haven't tested it on prod yet, so it may need to be tweaked further.  Otherwise this PR LGTM!

Reply:
Probably `"OSF Registries"` would be better. Prod shouldn't have an `_` in it, just the staging environments. And staging2 wouldn't have the `_` either, for reasons.

Reply:
This test doesn't run on prod IIRC. So `_OSF` should work and using `OSF` makes it eligible to be added to prod in the future.

Reply:
Great suggestion @felliott, we've added the changes.